backend/src/main/java/net/es/oscars/sb/nso/LiveStatusFdbCacheManager.java

```diff
@@ -0,0 +1,93 @@
+package net.es.oscars.sb.nso;
+
+import jakarta.transaction.Transactional;
+import lombok.extern.slf4j.Slf4j;
+import net.es.oscars.sb.nso.rest.DeviceServiceIdKeyPair;
+import net.es.oscars.sb.nso.rest.MacInfoServiceResult;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.stereotype.Component;
+
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@Component
+@EnableAsync
+public class LiveStatusFdbCacheManager {
+
+    final private NsoProxy nsoProxy;
+    private Map<DeviceServiceIdKeyPair, MacInfoServiceResult> cache;
+
+    public LiveStatusFdbCacheManager(NsoProxy nsoProxy) {
+        this.nsoProxy = nsoProxy;
+        this.cache = new HashMap<>();
+    }
+
+    @Transactional
```

Comment:
@Transactional is not needed anywhere in this class since you're not accessing any databases.

Reply:
isn't that also for concurrency? my understanding was that HashMap is not synchronized and that the annotation is providing the regarding isolation?
https://www.baeldung.com/spring-transactional-propagation-isolation#transactional-isolations

Reply:
Well - it doesn't work for you in this case because Hibernate could (and almost certainly does) allow multiple "transactions" to run in parallel. 

You could force serialization if you used  `@Transactional(isolation = Isolation.SERIALIZABLE)` but then you might as well make the method `synchronized`,  or used a `ConcurrentHashMap`.